JASP-Engine/JASP/R/descriptives.R

```diff
@@ -171,6 +171,9 @@ Descriptives <- function(jaspResults, dataset, options) {
     piePlots <- jaspResults[["pieCharts"]]
 
     for (var in variables) {
+      # skip non-categorical variables
```

Comment:
skipping all continuous variables - is there another way to find the variable specification from JASP? (I couldn't find it in "options")

Reply:
I don't think there is, this solution looks good to me.